packages/backend-modules/republik/script/finance/calculateKpis.js

```diff
@@ -135,7 +166,7 @@ const evaluateCompanyMonth = async (
         .filter((i) => i.companyName === company)
         .filter((i) => i.method === method)
         .filter((i) => i.type === 'MembershipType')
-        .filter((i) => ['CANCELLED', 'REFUNDED'].includes(i.status))
+        .filter((i) => ['CANCELLED', 'REFUNDED', 'failed', 'refunded'].includes(i.status))
```

Comment:
It would make sense define this filter function as a named function and use that instead of redefining six times.
I am also for normalising the case of `i.status` with `i.status.toUpperCase()` so that we don't have duplicated values.